photon-client/build.gradle

```diff
@@ -17,8 +17,7 @@ apply from: '../build-scripts/integration-test.gradle'
 
 dependencies {
   compile(project(':photon-avro-schemas'))
-  compile(project(":photon-lib$scalaSuffix"))
-  compile(project(":photon-api$scalaSuffix"))
+  compile(project(":photon-core$scalaSuffix"))
```

Comment:
I think you have an issue with your branch history, because you're undoing older commits in this one